specification/netapp/resource-manager/readme.md

```diff
@@ -334,3 +344,20 @@ terraform:
     clear-output-folder: false
 ```
 
+# Suppressions
+
+``` yaml
+suppressions:
+  - code: LroPostReturn
+    reason: The linter incorrectly flags LRO POST operations with no payload response, should allow to be a 202/204 NoContent for the LRO empty response case
+    from: netapp.json
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/backupVaults/{backupVaultName}/backups/{backupName}/restoreFiles"].post
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/capacityPools/{poolName}/volumes/{volumeName}/migrateBackups"].post      
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/migrateBackups"].post
+  - code: GetCollectionOnlyHasValueAndNextLink
```

Comment:
>GetCollectionOnlyHasValueAndNextLink [](http://example.com/codeflow?start=10&length=36)

The explanation for this suppression is unclear. pl elaborate

Reply:
Hi,  
This API was approved in a PR in the private repository in https://github.com/Azure/azure-rest-api-specs-pr/pull/8331 
There where dealys in getting it out but we are now bringing it now in prevew.
Fixing this response model according to the rule would requre changing the reponse which is something the team is trying to avoid this late in the game.

Reply:
@rkmanda  Would modeling this omething like  `/latestBackupStatus/current` suffice ?  
Will we need the collection GET as well ?

Reply:
Removed suppression added default name, `GetCollectionOnlyHasValueAndNextLink` does not show.

Reply:
Another option is to remove this from this version and introduce correctly modeled in the next. The preview clients have a workaround for this.  This would give the team the time to patch the service side code needed.